### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by 2375452377**
xxx

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ LOGGING = {
     'disable_existing_loggers': False,
     'formatters': {
         'verbose': {
-            # {lineno:3} 里的 :3 表示至少显示3格，{levelname:7} 同理 
+            # {lineno:3} 里的 :3 表示至少显示3格，{levelname:7} 同理。
```
Comment (Terrenceluo) [inline README.md]: abc

Comment (Terrenceluo) [inline README.md]: ssss

Comment (2375452377) [reply README.md]: efg

Comment (Terrenceluo) [inline README.md]: ww

Comment (Terrenceluo) [inline README.md]: fdsaf

Comment (2375452377) [reply README.md]: fd

**Pull request merged**